app/src/main/res/layout/settings_activity.xml

```diff
@@ -9,180 +9,197 @@
+                tools:srcCompat="@tools:sample/avatars[3]" />
+
+            <Button
+                android:id="@+id/button2"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="136dp"
+                android:layout_marginTop="1dp"
+                android:layout_marginEnd="136dp"
+                android:layout_marginBottom="409dp"
+                android:text="Change Picture"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/imageView3" />
+
+            <TextView
+                android:id="@+id/textView2"
+                android:layout_width="163dp"
+                android:layout_height="48dp"
+                android:layout_marginStart="16dp"
+                android:layout_marginTop="210dp"
+                android:layout_marginEnd="232dp"
+                android:layout_marginBottom="355dp"
+                android:text="Warrior Social"
+                android:textAppearance="@style/TextAppearance.AppCompat.Large"
+                android:textSize="18sp"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <Button
+                android:id="@+id/button3"
+                android:layout_width="182dp"
+                android:layout_height="55dp"
+                android:layout_marginStart="209dp"
+                android:layout_marginTop="206dp"
+                android:layout_marginEnd="20dp"
+                android:layout_marginBottom="352dp"
+                android:text="Change Username"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:id="@+id/textView"
+                android:layout_width="166dp"
+                android:layout_height="39dp"
+                android:layout_marginStart="16dp"
+                android:layout_marginTop="271dp"
+                android:layout_marginEnd="229dp"
+                android:layout_marginBottom="303dp"
+                android:text="***********"
+                android:textAppearance="@style/TextAppearance.AppCompat.Large"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <Button
+                android:id="@+id/button4"
```

Comment:
This should be the button linked to the issue above on line 51. Maybe consider switching this to id "resetPass" like you had originally since it makes it a little easier to follow when you have a descriptive id.